Sources/OpenTelemetrySdk/Trace/TraceConstants.swift

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020, OpenTelemetry Authors
```

Comment:
already 2021. My recommendation is just omit the year mentions across all files

Reply:
that's what I get for copy-pasting 😆